sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -6386,9 +6390,8 @@ public void onServiceStarted(SdlSession session, SessionType type, boolean isEnc
 		@Override
 		public void onServiceEnded(SdlSession session, SessionType type) {
 			if(SessionType.NAV.equals(type)){
-				dispose();
+				stopStreaming();

```

Comment:
Changing from `dispose()` to `stopStreaming()` introduces an unnecessary call to stop the video streaming service. `stopStreaming() -> ISDL.onServiceEnded() -> stopStreaming()` We should try to avoid this in this class rather than relying on downstream efforts. I would suggest either simply check if `remoteDisplay != null` or introduce a new flag that denotes if the `VideoStreamingManager` is currently streaming.